src/client/app/components/playlist/playlistForm.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Search from '../search';
+import SearchResults from '../search/searchResults';
+import SongList from './songList';
+
+const mapStateToProps = state => ({
+    auth: state.auth,
+    show: state.show,
+    playlist: state.playlist,
+    search: state.search,
+    nowPlaying: state.nowPlaying
+});
+
+const showCurrentPlaylist = (obj) => {
```

Comment:
Not sure what the internal structure of playlist or currentPlaylist is like but consider these other forms:
hasCurrentPlaylist(playlist)
isValidPlaylist(currentPlaylist)
shouldShowPlaylist(playlist)